superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -1372,12 +1372,11 @@ export function popStoredQuery(urlId) {
 export function popSavedQuery(saveQueryId) {
   return function (dispatch) {
     return SupersetClient.get({
-      endpoint: `/api/v1/saved_query/${saveQueryId}`,
+      endpoint: `/savedqueryviewapi/api/get/${saveQueryId}`,
```

Comment:
@jfrag1 thanks for the change. Is there any reason you opted for this approach as opposed to keeping the new endpoint and relaxing the `SavedQueryFilter` filter criterion? I guess the merit with the later is it's clearer how this works as opposed to the somewhat magical (and obscure) FAB REST API endpoint.

Reply:
@john-bodley The main reason I did it this way is that I'm pretty sure there are sevaral FAB-generated endpoints under the `SavedQueryRestApi` that utilize the `SavedQueryFilter` (set as the `base_filter` on the class), and I didn't want to change the behavior for those.  For example, the endpoint that fetches a list of a user's saved queries could start returning a list of all saved queries with that change.